profile.md

```diff
@@ -232,6 +229,37 @@ part of a Kubernetes API call:
   }
   ```
 
+## Bind Resource Object
+
+In the [Open Service Broker API specification](spec.md), requests to
+[create a Service Binding](spec.md#binding) can contain a `bind_resource`
+object in which Platforms MAY choose to add additional fields.
+
+#### Cloud Foundry Bind Resource Object
+
+The following properties are defined for usage within a Cloud Foundry
+deployment:
+
+- `space_guid`  
```

Comment:
Is this required or optional for CF?  We should be clear.

Reply:
optional - will make this clearer